examples/coalescing-bulkloader/src/main/java/com/github/benmanes/caffeine/examples/coalescing/bulkloader/CoalescingBulkloader.java

```diff
@@ -124,9 +129,9 @@
      *                       for the corresponding future.
      */
     public static <Key, Value, Intermediate> CoalescingBulkloader<Key, Value> byExtraction(int maxLoadSize, long maxDelay,
-            final Function<Intermediate, Key> keyExtractor,
-            final Function<Intermediate, Value> valueExtractor,
-            final Function<Stream<Key>, CompletableFuture<Stream<Intermediate>>> load) {
+                                                                                           final Function<Intermediate, Key> keyExtractor,
+                                                                                           final Function<Intermediate, Value> valueExtractor,
+                                                                                           final Function<Stream<Key>, CompletableFuture<Stream<Intermediate>>> load) {
```

Comment:
looks like a formatting mistake?

Reply:
oh nm, I see, to line up. just fyi we're loosely using Google's java style if that helps, but I am not particular on these examples.

Reply:
Not if you make the window big enough...
The formatter lines up everything after the opening bracket which is a bit far right in this case.

Reply:
I forced the parameters to separate lines